application/lib/catalog.dart

```diff
@@ -53,11 +54,64 @@ class _CatalogState extends State<Catalog> with SingleTickerProviderStateMixin {
             children: <Widget>[
               const Placeholder(color: Colors.blue),
               const Placeholder(color: Colors.teal),
-              Container(),
+              const ConsoleTab(),
             ],
           ),
         ),
       ],
     );
   }
 }
+
+class ConsoleTab extends StatefulWidget {
+  const ConsoleTab({Key key}) : super(key: key);
+  @override
+  _ConsoleTabState createState() => _ConsoleTabState();
+}
+
+class _ConsoleTabState extends State<ConsoleTab> {
+  TextEditingController _username;
+  TextEditingController _password;
+
+  @override
+  void initState() {
+    super.initState();
+    _username = TextEditingController();
+    _password = TextEditingController();
```

Comment:
we should also make sure to update the text controllers when the ServerDisposition changes